bench/commands/make.py

```diff
@@ -11,12 +11,13 @@
 @click.option('--frappe-branch', default=None, help="path to frappe repo")
 @click.option('--clone-from', default=None, help="copy repos from path")
 @click.option('--clone-without-update', is_flag=True, help="copy repos from path without update")
+@click.option('--clone-full', is_flag=True, help="make a full (deep) clone of the repositories")
 @click.option('--no-procfile', is_flag=True, help="Do not create a Procfile")
 @click.option('--no-backups',is_flag=True, help="Do not set up automatic periodic backups for all sites on this bench")
 @click.option('--skip-redis-config-generation', is_flag=True, help="Skip redis config generation if already specifying the common-site-config file")
 @click.option('--skip-assets',is_flag=True, default=False, help="Do not build assets")
 @click.option('--verbose',is_flag=True, help="Verbose output during install")
-def init(path, apps_path, frappe_path, frappe_branch, no_procfile, no_backups, clone_from, verbose, skip_redis_config_generation, clone_without_update, ignore_exist=False, skip_assets=False, python='python3'):
+def init(path, apps_path, frappe_path, frappe_branch, no_procfile, no_backups, clone_from, verbose, skip_redis_config_generation, clone_without_update, clone_full, ignore_exist=False, skip_assets=False, python='python3'):
```

Comment:
```suggestion
def init(path, apps_path, frappe_path, frappe_branch, no_procfile, no_backups, clone_from, verbose, skip_redis_config_generation, clone_without_update, clone_full=False, ignore_exist=False, skip_assets=False, python='python3'):
```

Setting a default would be nicer in case someone's using the function directly.

Reply:
Done